### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by 2045**
A suggestion how to add the mentioning of the standard on the homepage, under 'what we do'

### CODE REVIEW

**Comment by ElenaFdR** [comment]
I love this copy - it's totally ready to go. I'd still like to talk to our designer, @angelplasma, about placement.

**Comment by bvhme** [comment]
@elenaFdR can we merge this and then do design later?

**Comment by bvhme** [comment]
Although, seeing it checked out it totally messes up the design. I can see if I can come up with a short term fix for that.

**Comment by bvhme** [comment]
All right, I've designed it and put it in its own box: https://github.com/2045/publiccode.net/pull/1

Once we've integrated that I think it'd be good to go.

File: index.html [hunk 1]
```diff
@@ -44,6 +44,47 @@ <h2>
   </svg>
 </div>
 
+<div class="mp-section">
+
+  <div>
+    <h2>
+      <a href="https://standard.publiccode.net/">
+        We also manage & publish the Standard for Public Code.
```
Comment (ElenaFdR) [inline index.html:52]:
```suggestion
        We maintain the Standard for Public Code. 
```

File: index.html [hunk 2]
```diff
@@ -44,6 +44,47 @@ <h2>
...
 
+<div class="mp-section">
+
+  <div>
+    <h2>
+      <a href="https://standard.publiccode.net/">
+        We also manage & publish the Standard for Public Code. 
+      </a>
+    </h2>
+    <p>The Standard for Public Code gives public organizations a model for building their own open source solutions to enable successful future reuse by similar public organizations in other places. It includes guidance for policy makers, city administrators, developers and vendors.</p>
```
Comment (ElenaFdR) [inline index.html:55]:
```suggestion
    <p>The Standard for Public Code is a model for public organizations building open source solutions to enable successful reuse by other public organizations.</p>
```

**Review by ElenaFdR — APPROVED** [review]
This will be a super useful addition - thanks, @2045!

File: index.html [hunk 3]
```diff
@@ -44,6 +44,47 @@ <h2>
...
+<div class="mp-section">
+
+  <div>
+    <h2>
+      <a href="https://standard.publiccode.net/">
+        We maintain the Standard for Public Code
+      </a>
+    </h2>
+    <p>The Standard for Public Code is a model for public organizations building open source solutions to enable successful reuse by other public organizations.</p>
+    <p><a href="/https://standard.publiccode.net/">It is here for you to use.</a></p>
```
Comment (ElenaFdR) [inline index.html:56]:
```suggestion
    <p><a href="https://standard.publiccode.net/">It is here for you to use.</a></p>
```

**Pull request merged**